themes/ansible-community/templates/base.tmpl

```diff
@@ -9,12 +9,16 @@
 {% endblock %}
 {{ template_hooks['extra_head']() }}
 </head>
-<body id="top">
+<body id="top" class="{% if permalink == '/' %}homepage{% endif %}">
```

Comment:
this leaves and empty "class" parameter on other pages. Probably not harmful, but if we really want no class here, shouldn't the `class=` move inside the if?

Reply:
oh, nice catch.